### ISSUE TITLE
please update proxy-agent@3.x to 4.x to mitigate netmask vulnerability (CVE-2021-28918 / CWE-20)

### ISSUE THREAD

**Issue by mashpie** [keymetrics/pm2-io-agent-node#17]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ netmask npm package vulnerable to octal input data           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ netmask                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.0.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ pm2                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ pm2 > @pm2/io > @pm2/agent-node > proxy-agent >              │
│               │ pac-proxy-agent > pac-resolver > netmask                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1658                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

The dependency tree is fixed up until proxy-agent and should finally refer to fixed netmask version:
https://github.com/TooTallNate/node-pac-resolver/pull/25

pm2-io-agent-node pins proxy-agent to 3.x but 4.x is required to resolve upgrades with current patches.

**Comment by Unitech** [comment]
Fixed via pm2@4.5.6

```bash
npm install pm2@latest -g
pm2 update
```

**Comment by mashpie** [comment]
Thanks! Nice! Works!
